app/labor/assign_tag_moderator.rb

```diff
@@ -1,63 +1,73 @@
 module AssignTagModerator
-  def self.add_trusted_role(user)
-    return if user.has_role?(:trusted)
-    return if user.has_role?(:banned)
-
-    user.add_role :trusted
-    user.update(email_community_mod_newsletter: true)
-    MailchimpBot.new(user).manage_community_moderator_list
-    Rails.cache.delete("user-#{user.id}/has_trusted_role")
-    NotifyMailer.with(user: user).trusted_role_email.deliver_now
-  end
+  class << self
```

Comment:
Personally, not a big fan of this form. It's more obscure to people new to Ruby and adds an extra level of indentation. I prefer the most explicit form, which is the one this PR changes. If we don't want that for whatever reason, `extend self` at least doesn't add extra identation and requires roughly the same level of Ruby knowledge as `class << self`:

```ruby
module Test
  extend self

  def test
    "Hi, Andy! 👋"
  end
end

Test.test
#=> "Hi, Andy! 👋"

Test.methods - Object.methods
#=> [:test]
```

Reply:
Ah gotcha. I'll revert it back, good point about `self.method` probably being easier to understand for newcomers to Ruby.